django-workload/django_workload/bundle_tray.py

```diff
@@ -53,34 +53,38 @@ def get_bundle(self):
         return result
 
     def get_inc_factor(self):
-        global INC_FACTOR
-        return int((INC_FACTOR + 1) / 2)
-
-    def post_process(self, res):
-        bundle_list = res['bundle']
-        conf = BundleConfig()
+        return int((INC_FACTOR + 1 + 1) / 3)
```

Comment:
Is and extra +1 (why not +2) and divide by 3 here intentional - Old code was different

Reply:
After refactoring the code and removing EXTENDED_ARG, the bytecode histogram changed slightly and I needed to boost LOAD_CONST usage in order for the new histogram to remain representative. That is why I added a +1 to the original code.